bugbug/db.py

```diff
@@ -60,7 +49,7 @@ def download_support_file(path, file_name):
         utils.download_check_etag(url, path)
 
         if path.endswith(".zst"):
-            extract_file(path)
+            utils.zstd_decompress(path)
```

Comment:
You are passing a path with the `.zst` extension here, and without the extension at https://github.com/mozilla/bugbug/pull/790/files#diff-2f714e5a8f0bf3c9fa77453ee08b8ddaR79.